SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/HomeViewController.swift

```diff
@@ -206,4 +208,13 @@ extension HomeViewController: SliderListDelegate{
         self.collectionView.reloadData()
     }
     
+    func loading(state: Bool) {
```

Comment:
this is anti intuitive  loading false leads to show the spinner the naming is important so the next programer knows what happens before read the function please put a more  something understandable